packages/core/content-releases/package.json

```diff
@@ -84,6 +85,7 @@
     "typescript": "5.2.2"
   },
   "peerDependencies": {
+    "@strapi/admin": "^4.15.1",
```

Comment:
This needs to be 4.19.0, next version we're releasing & the first version that has the hook exported!